integration-cli/docker_cli_external_graphdriver_unix_test.go

```diff
@@ -285,21 +283,19 @@ func (s *DockerExternalGraphdriverSuite) SetUpSuite(c *check.C) {
 		respond(w, &graphDriverResponse{Size: size})
 	})
 
-	if err := os.MkdirAll("/etc/docker/plugins", 0755); err != nil {
-		c.Fatal(err)
-	}
+	err = os.MkdirAll("/etc/docker/plugins", 0755)
+	c.Assert(err, check.IsNil, check.Commentf("error creating /etc/docker/plugins"))
 
-	if err := ioutil.WriteFile("/etc/docker/plugins/test-external-graph-driver.spec", []byte(s.server.URL), 0644); err != nil {
-		c.Fatal(err)
-	}
+	err = ioutil.WriteFile("/etc/docker/plugins/test-external-graph-driver.spec", []byte(s.server.URL), 0644)
+	c.Assert(err, check.IsNil,
```

Comment:
Do we really need to break these to two-lines? It is recommend in C, but not sure if it was the same for Golang